win32ss/gdi/ntgdi/gdidebug.h

```diff
@@ -125,11 +125,10 @@ GdiDbgPostServiceHook(ULONG ulSyscallId, ULONG_PTR ulResult);
 #define ID_Win32PreServiceHook 'WSH0'
 #define ID_Win32PostServiceHook 'WSH1'
 
-#ifndef __cplusplus
```

Comment:
Why would GdiDbgAssertNoLocks be disabled?

Reply:
The return of `PsGetCurrentThreadWin32Thread` function is `PVOID`.
C++ won't implicitly type cast `PVOID`.